### PR TITLE
updated the home page...

### LINKED ISSUE #4: Make a home page with a description of website (including an image)
use the design template we made in figma ..  
link to the project in figma = https://www.figma.com/file/9REOLbmuv5SqJKNBly7JIH/Untitled?node-id=0%3A1

### PR DESCRIPTION
**Pull request by HosenAgbaria**
updated the home page and copyed omri's code in the "Render search data" pull requset to make sure it fits on the new home page...
closes #4

### CODE REVIEW

**Review by jamiecoe — CHANGES_REQUESTED** [review]
Before merging this PR, we should merge Omri's Render Search Data PR #31 - I've requested some changes so these just need to be fixed first. 

The additional stuff here looks good apart from the formatting!! Please please start using a formatting tool like Prettier, it makes the code so much easier to read.

Fix the formatting and then this PR will be ready to merge once #31 is merged.

File: the-experts-fe/src/App.js
Comment (jamiecoe) [inline the-experts-fe/src/App.js]: Remember to use `res.json()` instead of `res.text()` here as the content-type of the response is JSON.

File: the-experts-fe/src/App.css
Comment (jamiecoe) [inline the-experts-fe/src/App.css]: Still have dodgy formatting here! Use Prettier 💅

File: the-experts-fe/src/App.js
Comment (jamiecoe) [inline the-experts-fe/src/App.js]: Dodgy formatting! Use Prettier 💅

**Review by jamiecoe — CHANGES_REQUESTED** [review]

**Review by omriza5 — APPROVED** [review]

**Review by ebraheemabbas — APPROVED** [review]

**Review by jamiecoe — APPROVED** [review]

**Pull request merged**
